ASM/src/config.asm

```diff
@@ -123,6 +123,11 @@ WORLD_STRING_TXT:
 TIME_STRING_TXT:
 .endarea
 
+; web seed ID string (max length 10 chars)
+.area 0x10, 0
```

Comment:
Since it's 10 chars should this be 0x0A?

Reply:
`0x0B` technically, to account for the null terminator. I just made it match the `WORLD_STRING_TXT` symbol above which is also 10 chars.

Reply:
Fair enough then